crates/bevy_pbr/src/render/pbr_debug.wgsl

```diff
@@ -0,0 +1,245 @@
+#define_import_path bevy_pbr::pbr_debug
+
+#ifdef PBR_DEBUG
+
+// Used when debug stuff is missing
+var<private> pink: vec4<f32> = vec4<f32>(1.0, 0.0, 1.0, 1.0);
```

Comment:
It would be nice if that was configurable, but that's probably going to be a lot more annoying to setup, so probably leave it as is for now.

Reply:
It feels like not a big deal. It's only for debug purposes. :)